File: HTTP_Server_CLI/scenarios/1.3.1-catch_all.lua
-- @label: Catch-all 404
-- @group: Catch-all
-- @competitors: all
-- All requests hit non-existent paths, triggering the catch-all handler.

local counter = 0

request = function()
   counter = counter + 1
   return wrk.format("GET", "/not-found-" .. (counter % 1000))
end

File: HTTP_Server_CLI/scenarios/php/1.3.1-catch_all.php
<?php
// @label: Catch-all 404
// @group: Catch-all
// @competitors: all
// All requests hit non-existent paths, triggering the catch-all handler.

return [
   'method' => 'GET',
   'paths'  => array_map(fn ($i) => "/not-found-{$i}", range(0, 999)),
];
